libraries/chef_pyenv_script_helpers.rb

```diff
@@ -5,12 +5,12 @@ module Pyenv
     module ScriptHelpers
       include Chef::Mixin::ShellOut
       def root_path
-        node.run_state['root_path'] ||= {}
+        node.run_state['sous-chefs']['pyenv']['root_path'] ||= {}
```

Comment:
I am pretty sure you have to initialise each level in `run_state` so you will need to do this:

```
         node.run_state['sous-chefs'] ||= {} 
         node.run_state['sous-chefs']['pyenv'] ||= {} 
         node.run_state['sous-chefs']['pyenv']['root_path'] ||= {} 
```

Reply:
Yeah I just realized that